AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Images-out-of-focus.rst

```diff
@@ -0,0 +1,117 @@
+.. This is a template for Image out of focus when some part of the observatory enters an abnormal state. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Carlos Morales, Karla Aubel, Erik Dennihy*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-AuxTel-Troubleshooting-General-Troubleshooting-AuxTel-Image-out-of-focus-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+AuxTel Image out of focus
+#########################
+
+.. _Image-out-of-focus-Procedure-Overview:
+
+Overview
+========
+
+This procedure should be used when images appear out of focus due to a *WEP failure* or other related issues. Users will notice images ranging from slightly out of focus to large donut appearances when hexapod offsets are not correctly applied.
```

Comment:
1. WEP Failure in bold not italics
2. hexapods --> ATHexapods

Reply:
Italics can also be used for concepts or remarks, to avoid overusing of bold as well. I left it "WEP failure" in italics here, but may consolidate in the future in a different format. Thanks for noticing.